ecs-cli/modules/commands/flags.go

```diff
@@ -77,13 +79,30 @@ const (
 	RoleFlag                                = "role"
 )
 
+// OptionalClusterAndProfileFlags provides these flags:
 // OptionalClusterFlag inline overrides cluster
-func OptionalClusterFlag() cli.Flag {
-	return cli.StringFlag{
-		Name: ClusterFlag + ", c",
-		Usage: fmt.Sprintf(
-			"[Optional] Specifies the ECS cluster name to use. Defaults to the cluster configured using the configure command",
-		),
+// OptionalClusterConfigFlag specifies the cluster profile to read from config
+// OptionalProfileConfigFlag specifies the credentials profile to read from the config
```

Comment:
So this code is just defining where the flags are valid. And yeah anywhere where the cluster and profile config flags are valid, the cluster flag will be as well.